packages/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.9+2
+
+* Remove 'init' call to avoid MissingPluginException when testing.
```

Comment:
This description is hard to read without more context.
 
I don't think the init call is exposed in the public api, and therefore its hard for an end-user to know what are the consequences of  the change.

Also it is not obvious why having the init-call leads to a MissingPluginException when testing